src/main/java/com/spikes2212/command/LEDs.java

```diff
@@ -0,0 +1,94 @@
+package com.spikes2212.command;
+
+import com.spikes2212.dashboard.Namespace;
+import com.spikes2212.dashboard.RootNamespace;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+
+import java.awt.*;
+
+/**
+ * Class that gives more control over LEDs.
+ *
+ * @author Camellia Lami
+ * @see DashboardedSubsystem
+ */
+public class LEDs extends DashboardedSubsystem {
+
+    /**
+     * Creates a {@link AddressableLED} and a {@link AddressableLEDBuffer}.
+     */
+    private final AddressableLED led;
+    private final AddressableLEDBuffer ledBuffer;
+
+    public LEDs(Namespace namespace, AddressableLED led, AddressableLEDBuffer ledBuffer) {
```

Comment:
don't get the leds and the addressable led buffer, instead get the led port and the number of leds and create the objects in the constructor